packages/app/src/AppNav.tsx

```diff
@@ -514,6 +517,27 @@ export default function AppNav({ fixed = false }: { fixed?: boolean }) {
         : 'ok' // All alerts are green
       : 'none'; // No alerts are set up
 
+  const [searchesListQ, setSearchesListQ] = useState('');
```

Comment:
Could consider using local storage here and for `dashboardsListQ`, too.

Reply:
was thinking about it, but not sure if keeping the search query after page refresh it a good ux. on the other hand, there's this bug with AppNav being completely re-mounted and resetting everything when moving between pages – i'm going to address it separately, should be something with next.js templates